build-binutils.py

```diff
@@ -77,7 +77,7 @@ def parse_parameters(root_folder):
                         help="""
                         The script can build binutils targeting arm-linux-gnueabi, aarch64-linux-gnu,
                         mipsel-linux-gnu, powerpc-linux-gnu, powerpc64-linux-gnu, powerpc64le-linux-gnu,
-                        and x86_64-linux-gnu.
+                        s390x-linux-gnu, and x86_64-linux-gnu.
```

Comment:
RISC-V is missing here.

Reply:
Thanks, 5ddb83db1eeae4ca6e75db645afb288b20ea9bba